src/sentry/middleware/dedupe_cookies.py

```diff
@@ -67,6 +67,29 @@ class DedupeCookiesMiddleware:
 
     def __init__(self, get_response: Callable[[Request], Response]):
         self.get_response = get_response
+        self.COOKIE_DOMAINS = {
+            settings.SESSION_COOKIE_NAME: getattr(settings, "SESSION_COOKIE_DOMAIN", None),
+            settings.CSRF_COOKIE_NAME: getattr(settings, "CSRF_COOKIE_DOMAIN", None),
+            superuser.COOKIE_NAME: superuser.COOKIE_DOMAIN,
+            sudo_settings.COOKIE_NAME: sudo_settings.COOKIE_DOMAIN,
+        }
+
+    def pick_domain(self, cookie_name: str, request: Request) -> str:
+        # Pick a domain to delete the cookie from.
+        domain = "sentry.io"
```

Comment:
Will this hardcoded value do the wrong thing for single-tenants? Should we be using system.url-prefix instead?